src/Events/Swagger/RequestBodyExampleFilter.cs

```diff
@@ -41,6 +41,8 @@ private static void CreateCloudEventRequestModelExamples(OpenApiRequestBody requ
                 ("Instance created event with alternative subject",
                 CreateOpenApiObject(new List<(string Name, string Value)>()
                 {
+                    ("resource", "urn:altinn:resource:altinnapp.ttd.apps-test"),
```

Comment:
Should we update the example for the Generic POST event endpoint as well?

Reply:
the generic post example is based on the cloud event schema. Added resource prop to that schema as well. We don't have a set of example request bodies as we have for other endpoints.